charts/stable/valheim/questions.yaml

```
@@ -9,205 +9,201 @@ questions:
 # Include{replica1}
 # Include{controllerExpertExtraArgs}
   - variable: secretEnv
-    group: "App Configuration"
-    label: "Image Secrets"
+    group: App Configuration
+    label: Image Secrets
     schema:
       additional_attrs: true
       type: dict
       attrs:
         - variable: SUPERVISOR_HTTP_USER
-          label: "SUPERVISOR_HTTP_USER"
+          label: SUPERVISOR_HTTP_USER
           schema:
             type: string
-            default: "admin"
+            default: admin
```

```suggestion
            default: ""
```

Let's have those empty so users explicitly set username/pass